L10n/de.strings

```diff
@@ -1,44 +1,47 @@
 /* Language name */
 "!Language" = "Deutsch";
 
+/* template for ship name */
+" with " = "Deutschverb"
+
 /* Idle message. [load.py] */
 "Connecting CMDR Interface" = "Verbindet mit KMDT-Schnittstelle";
 
 /* Details of system. [load.py] */
-"In system {system}" = "Im System {system}";
+"In system {system}{ship}" = "Im System {system}{ship}";
```

Comment:
I have seen that the ship name is right next to other strings without any spaces. Can you make sure that there is proper spacing in all places?
Maybe do something like `In system {system} flying {ship}` (only if the length aint too much)

Reply:
That was a mistake on my part when I wrote the find and replace command, fixing with next commit